Index: app/src/main/java/com/example/appgps/MainActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/appgps/MainActivity.kt b/app/src/main/java/com/example/appgps/MainActivity.kt
--- a/app/src/main/java/com/example/appgps/MainActivity.kt	
+++ b/app/src/main/java/com/example/appgps/MainActivity.kt	(date 1686951393096)
@@ -123,7 +123,7 @@
 
         val atleta = "Bruno"
         val timestamp = System.currentTimeMillis()
-        val sensorData = SensorData("\n" + atleta, latitude, longitude,+"\n" + x, y, z, timestamp)
+        val sensorData = SensorData(atleta, latitude, longitude, x, y, z, timestamp)
         sensorDataList.add(sensorData)
 
         adapter?.notifyDataSetChanged()
@@ -212,7 +212,7 @@
 
         val atleta = "BRUNO"
         val timestamp = System.currentTimeMillis()
-        val sensorData = SensorData("\n" + atleta, latitude, longitude,+"\n" + x, y, z, timestamp)
+        val sensorData = SensorData(atleta, latitude, longitude, x, y, z, timestamp)
         sensorDataList.add(sensorData)
 
         adapter?.notifyDataSetChanged()
